src/net.cpp

```diff
@@ -605,12 +605,12 @@ void CNode::CopyStats(CNodeStats& stats)
     X(m_bip152_highbandwidth_from);
     {
         LOCK(cs_vSend);
-        X(mapSendBytesPerMsgCmd);
+        X(mapSendBytesPerMsg);
```

Comment:
Shouldn't `mapSendBytesPerMsgCmd` be renamed to `mapSendBytesPerMsgType`?

Reply:
I think this would be a better naming convention for the variables.

Addressed this and the following comments in 8f31aadd25ad434536ba0d166b81324a18180acb